dsgrid/config/annual_time_dimension_config.py

```diff
@@ -49,7 +51,20 @@ def check_dataset_time_consistency(self, load_data_df):
                 f"load_data {time_col}s do not match expected times. mismatch={mismatch}"
             )
 
-    def convert_dataframe(self, df, project_time_dim):
+    def get_time_dataframe(self):
```

Comment:
This method should probably be defined as an abstractmethod in the base class.

Reply:
add to base class